### DISCUSSION TITLE
1.6.2

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#1286] [New Release]
### Changelog
- Fix resource hook in Android 12 release
- Fix modules not working in the early period after reboot
- Show updates and shortcut request only when activated
- Fix crash of parasitic manager on some devices
- Fold long module descriptions

### 更新日志
- 修复安卓 12 正式版的资源挂钩
- 修复模块刚开机时不能激活
- 只在激活情况下显示更新和快捷方式请求
- 修复寄生管理器在某些设备下闪退
- 折叠过长的模块描述

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.6.2'>1.6.2</a>.</em>

**Comment by wzcnm** [comment]
从1.6.0开始，在OPPO手机ColorOS 7安卓10上，用面具刷入后，开机无法进入桌面，导致无限重启。这个问题是否考虑解决？在ColorOS 11安卓11上就可以正常激活使用！

**Reply by yujincheng08** [reply]
收到过类似的报告，但是没有一个人提供日志，只能说爱莫能助。

**Reply by jtmyd-top** [reply]
为什么我的lsp模块也安装了就是不能激活

**Comment by whx9986** [comment]
反馈：lsp仓库的杜比大喇叭β项目地址改了，目前已更新到3.2.1，但lsp仓库仍是2.1.0，地址应该是https://github.com/nining377/dolby_beta

**Reply by JasonKhew96** [reply]
~~作者偷懒，没有传仓库~~
